inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape.cpp

```diff
@@ -103,7 +103,7 @@ const Transformations& getDefaultTransformations() {
 }
 
 std::set<NodeTypeInfo> getSupportedTypes(const Transformations& transformations) {
-    auto supportedTypes = std::set<NodeTypeInfo>{};
+    std::set<NodeTypeInfo> supportedTypes{};
```

Comment:
And here

Reply:
> 'supportedTypes' is used uninitialized in this function

Reply:
It really looks like a false-positive

Reply:
I think so, so I'd prefer to mark it as FP if possible

Reply:
Since you pushed anyway you could fix that properly..